### PR TITLE
Part 1 of RFC for SQL Support

### PR DESCRIPTION
**Pull request by kevinhinterlong**
Initialize fili-sql module
@michael-mclawhorn @archolewa 

RFC: #475

### CODE REVIEW

File: fili-sql/src/main/resources/moduleConfig.properties [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+bard__request_limit_global = 70
+bard__request_limit_per_user = 2
+bard__request_limit_ui = 52
+
+# setting for druid query weight limit
+bard__query_weight_limit = 100000
+
+# Default the timeout to 10 minutes, in milliseconds
+bard__druid_request_timeout = 600000
+
```
Comment (michael-mclawhorn) [inline fili-sql/src/main/resources/moduleConfig.properties]: Clean out config that isn't specific to your module.
Add module dependency to fili-core

Comment (kevinhinterlong) [inline fili-sql/src/main/resources/moduleConfig.properties]: Updated

File: fili-sql/src/main/resources/moduleConfig.properties [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+#The name of this module (required)
+moduleName = fili-sql
+
+moduleDependencies = fili-core
+
+# Disabling the druid dimension loader by default
+# When set to true you will also need to populate the list of dimensions using druid_dim_loader_dimensions
+bard__druid_dimensions_loader_enabled = false
+bard__druid_dim_loader_row_limit = 1000
```
Comment (kevinhinterlong) [inline fili-sql/src/main/resources/moduleConfig.properties]: These aren't relevant until later in the rfc, should I take it out @michael-mclawhorn ?

Comment (michael-mclawhorn) [inline fili-sql/src/main/resources/moduleConfig.properties]: Nah. It'll arrive in time.

**Pull request merged**